psi4/driver/procrouting/proc_util.py

```diff
@@ -221,4 +221,4 @@ def print_ci_results(ciwfn, rname, scf_e, ci_e, print_opdm_no=False):
         ciwfn.print_vector(dvec, root)
 
     # True to keep the file
-    #dvec.close_io_files(True)
+    dvec.close_io_files(True)
```

Comment:
Can't believe it was something so simple.  I was searching for something screwy in libtrans